Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/workspace/ListWSManager.java

```diff
@@ -263,8 +263,10 @@ public String getWorkspaceName(String workspaceId) throws QuadrigaStorageExcepti
                 }

```

Comment:
in line 250, negate the condition and move the code from the else block into the if block.